src/olympia/reviewers/utils.py

```diff
@@ -919,6 +923,7 @@ def get_context_data(self):
             ),
             'comments': self.data.get('comments'),
             'SITE_URL': settings.SITE_URL,
+            'reasons': self.data.get('reasons'),
```

Comment:
`get_context_data` is used for the email (I don't see any changes to the email template)

Reply:
I thought it was used for the form, so when validations fail the form can be re-displayed with the data. Is that not the case?

Reply:
A quick test shows me that I was wrong. :-)

I'll remove this.

Reply:
not as far as I can see - it's the context for the email template - but reviewer tools is a sprawling chunk of code so I wouldn't be too surprised if I was wrong.